karma.conf.js

```diff
@@ -1,33 +1,39 @@
-module.exports = function (config) {
-    config.set({
-        browsers: ['PhantomJS'],
+module.exports = (config) => {
```

Comment:
If you are using ES6, should you just `export default` this? Also, it looks like Travis is not responding properly to the ES6 change here, is there something you need to update to make that work?